src/lib.rs

```diff
@@ -304,11 +361,36 @@ pub fn eval<R, F: FnOnce(ReturnValue) -> R>(name: &str, f: F) -> Option<R> {
     p.eval(name).map(f)
 }
 
+/// Set new actions to a fail point at runtime.
+///
+/// Actions should be in the format of `action[->action...]`. When multiple action
```

Comment:
The format of the actions is: `action[->action...]`. When multiple actions are specified, an action will be checked only if its former action is not triggered.